tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/asynchronous/AsyncClassLevelClient.java

```diff
@@ -41,28 +41,6 @@
 @Asynchronous
 public class AsyncClassLevelClient {
 
-
-    /**
-     * service 1 second in normal operation.
-     * @return the result as a Future
-     * @throws InterruptedException the interrupted exception
-     */
-    public Future<Connection> service() throws InterruptedException {
-
-        Connection conn = new Connection() {
-            {
-                Thread.sleep(1000);
-            }
-
-            @Override
-            public String getData() {
-                return "service DATA";
-            }
-        };
-
-        return CompletableFuture.completedFuture(conn);
-    }
-    
```

Comment:
Unless we're about to remove support for `Future` as a return type of `@Asynchronous` methods (which I'd totally agree with :-) ), I think we need both `service` and `serviceCS`. It should be possible to rewrite `service` to accept a `waitCondition` similarly to `serviceCS`.